### ISSUE TITLE
Documentation or running installer without cloud shell is incorrect

### ISSUE THREAD

**Issue by lukwam** [GoogleCloudPlatform/forseti-security#1293]
Version 2rc1 currently says:

```
1) Create a project.
2) Enable billing for the project.
3) Install gcloud and authenticate your account using "gcloud auth login".
4) Set your project using "gcloud config project set <PROJECT_ID>".
5) Run this setup again, with the --no-cloudshell flag, i.e.

        python setup_forseti.py --no-cloudshell
```

However, it should probably be `python setup/installer.py --no-cloudshell` as there is no file called `setup_forseti.py` as far as I can see.

**Comment by mirons-google** [comment]
HI @lukwam Let me know if I'm wrong, but I believe you might be mixing in `v1.X` docs on forsetisecurity.org with `2.0RC` [installation](https://docs.google.com/document/d/1RDw8QLhJVd-EAwIAviaboXwLoHPehteRGLCOXzEHBoc/preview) and [testing](https://docs.google.com/document/d/1e6uJgTZTtTCMFaZ0BMs_EtV9ULcunq-tKsNiSxIMg3M/preview#heading=h.jvgcd8ooswz6) guides.

Those two guides will be the basis for the `2.0` doc changes.

**Comment by lukwam** [comment]
I am running the installer in 2rc1 and when you run it it has output that tells you how to run it with --no-cloudshell but it gives the incorrect pasth to the setup.py in the output .

https://github.com/GoogleCloudPlatform/forseti-security/blob/f2dbd40eaecbd0bbe95101e930c6dad84f6f0b03/scripts/gcp_setup/environment/gcloud_env.py#L335

**Comment by mirons-google** [comment]
Ah, cheers! Thanks for reporting the issue. @joecheuk will fix this in #1303 .

**Comment by joecheuk** [comment]
Resolved in #1303
